### PR TITLE
Add Getting Started with Redis OM Python tutorial

### PR DESCRIPTION
**Pull request by abrookins**
Add a tutorial for Redis OM for Python.

### CODE REVIEW

**Comment by simonprickett** [comment]
@abrookins ping me when this is ready and I'll take a look, thanks!

**Comment by abrookins** [comment]
@simonprickett This is ready to review!

File: docs/develop/python/redis-om/index-redis-om.mdx [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+use.
+
+**TIP:** The `-d` option in these examples runs Redis in the background, while 
+`-p 6379:6379` makes Redis reachable at port 6379 on your localhost.
+
+#### Docker with the `redismod` image (recommended)
+
+    $ docker run -d -p 6379:6379 redislabs/redismod
+
+### Docker with the `redis` image
```
Comment (simonprickett) [inline docs/develop/python/redis-om/index-redis-om.mdx]:
```suggestion
#### Docker with the `redis` image
```

Lines up heading with the preceding one.

**Review by simonprickett — APPROVED** [review]
@abrookins looks good to me - left one small suggested commit to adjust the level of one heading to be in line with its sibling in the Docker section.  My only other thought was do you want to add a link to the Redis OM Python GitHub repo somewhere in this article, I don't think I saw one.

**Comment by simonprickett** [comment]
@abrookins FYI I updated the status here from draft to ready for review.  I reviewed it, I think you're good to go.

**Comment by simonprickett** [comment]
@abrookins I fixed the merge conflict with the authors list in the docusaurus config file.

**Comment by abrookins** [comment]
Thanks for your sharp eye and all your help, @simonprickett !

**Pull request merged**